README.md

```diff
@@ -43,13 +43,17 @@ The following lines can be used as swappy's default:
 ```
 [Default]
 save_dir=$HOME/Desktop
+save_filename=swappy-#.png
+save_filename_fmt=%Y%m%d-%H%M%S
```

Comment:
Hey  @apiraino do you think you could find a way to only have one setting for both? Somehow merging `save_filename` into the `save_filename_fmt` settings. Also I prefer explicit naming rather than short ones so `save_filename_format` would be better.

Reply:
I agree, it makes more sense and it's cleaner. Will do in the next couple of days